site/learn/Learn-Execution.md

```diff
@@ -149,6 +153,52 @@ The resolver for this field is not just returning a Promise, it's returning a *l
 GraphQL will wait for all of these Promises concurrently before continuing, and when left with a list of objects, it will concurrently continue yet again to load the `name` field on each of these items.
 
 
+## Providing extra data in a resolver
+
+The resolver returns next reducer's root value. Traditionally, this root value would match the instance of the Type resolved.
+
+```js
+Query: {
+  human(obj, args, context) {
+    return context.db.loadHumanByID(args.id).then(
+      userData => new Human(userData)
+    )
+  }
+}
+```
+
+But nothing prevents you for returning extra elements.
+For instance, returning in a reducer a pointer to its parent may be usefull.
```

Comment:
This is an anti pattern that we should not encourage in the docs

Reply:
Hey @leebyron. What is the alternative to modifying the parent value? I'm having a really hard time finding any workaround for this online.

Reply:
@felipap It seems that the anti pattern here is creating context dependent child resolvers.